internal/commands/vaultsecrets/apps/create.go

```diff
@@ -106,6 +103,13 @@ func createRun(opts *CreateOpts) error {
 
 	if opts.Output.GetFormat() == format.Unset {
 		fmt.Fprintf(opts.IO.Err(), "%s Successfully created application with name %q\n", opts.IO.ColorScheme().SuccessIcon(), opts.AppName)
+
+		command := "$ hcp vault-secrets secrets create <secret name> --data-file <path to secret>"
+
+		fmt.Fprintln(opts.IO.Err())
```

Comment:
is it easy to use heredoc here?

Reply:
Didn't try with heredoc, but figured this approach is more consistent other implementations in the codebase ([here](https://github.com/hashicorp/hcp/blob/9d3adac44cf8431bd5ea9e53b3fa53fdf9548261/internal/commands/iam/workloadidentityproviders/create_aws.go#L187-L196) as an example).

Reply:
Sounds good to me if an existing usage pattern in the repo. I wasn't sure, but feel free to close this out. Thanks!